rageval/utils/check_utils.py

```diff
@@ -21,13 +21,13 @@ def text_to_sents(text: str, model_name="nltk") -> List[str]:
         sentences = [s.strip() for s in sentences if len(s.strip()) >= 3]
 
     elif model_name == "gpt-3.5-turbo":
-        model = OpenAILLM("gpt-3.5-turbo-16k", "OPENAI_API_KEY")
-        prompt = DOC_TO_SENTENCES_PROMPT
-        input_str = prompt.format(doc=text).strip()
-        r = model.generate([input_str])
-        sentences = eval(r)
+        model = OpenAILLM("gpt-3.5-turbo-16k", "OPENAI_API_KEY")  # pragma: no cover
```

Comment:
"gpt-3.5-turbo-16k" has been deprecated few months ago, we can use "gpt-3.5-turbo" instead.